geonode/documents/templates/layouts/doc_panels.html

```diff
@@ -286,7 +286,7 @@
                                         const image = document.getElementById('image');
                                         const path = location.pathname.split('/');
                                         const id = path[path.length - 2];
-                                        const prefetch_url = "{{ resource.curatedthumbnail.thumbnail_url }}";
+                                        const prefetch_url = "{{ resource.thumbnail_url }}";
```

Comment:
Same as above, are we sure we are not breaking the thumbnail upload? Why this is not coherent with Layers, Maps and Apps too? If the curated_thumbnail won't be used anymore, we should remove it form the "base" model too.

Reply:
> Same as above, are we sure we are not breaking the thumbnail upload? Why this is not coherent with Layers, Maps and Apps too? If the curated_thumbnail won't be used anymore, we should remove it from the "base" model too.

I'm investigating it but looks like that for the layer & maps, the `thumbnail_url` is already called instead of the `curatedthumbnail`, for apps the update is in this PR